docs/docs/concepts/functional_api.md

```diff
@@ -1,15 +1,15 @@
 # Functional API
 
 !!! warning "Beta"
-    The Functional API is currently in **beta** and is subject to change. Please [report any issues](https://github.com/langchain-ai/langgraph/issues) or feedback to the LangGraph team. Use the label `#functional-api` when creating issues in Github.
+    The Functional API is currently in **beta** and is subject to change. Please [report any issues](https://github.com/langchain-ai/langgraph/issues) or feedback to the LangGraph team.
 
 ## Overview
 
-LangGraph's **Functional API** allows using standard control flow constructs (e.g., loops, conditionals) to define complex workflows without having to define a graph.
+The Functional API is an alternative to [State Graph](low_level.md#stategraph) for development in LangGraph. 
```

Comment:
should this be "Graph API" for consistency? or we should use "State Graph" everywhere